spec/spec_helper.rb

```diff
@@ -16,7 +16,22 @@
 # users commonly want.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+
+require 'vcr'
+real_requests = ENV["REAL_REQUESTS"]
+VCR.configure do |c|
+  c.hook_into :webmock
+  c.cassette_library_dir = 'spec/support/vcr_cassettes'
+  c.configure_rspec_metadata!
+  c.allow_http_connections_when_no_cassette = true if real_requests
+  c.default_cassette_options = {:record => :new_episodes}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.